suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
```

Comment:
Yeah good point.  Worth a temporary variable to ensure the same path is used in both places:

    rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
    if [ -e $rabbitmqctl ]; then
        ....
        su rabbitmq -s /bin/sh -c "$rabbitmqctl report" > ${rabbitmq_report_log}
        ....